disco_aws_automation/resource_helper.py

```diff
@@ -211,3 +208,13 @@ def check_written_s3(object_name, expected_written_length, written_length):
         raise S3WritingError(
             "{0} is not written correctly to S3 bucket".format(object_name)
         )
+
+
+def backoff(cycle):
+    """This function takes as input an integer that represents a cycle count,
```

Comment:
For consistency with the rest of the project, this should be spread over two lines, like so:
```
"""
This function takes as input an integer that represents a cycle count,
```